promql/functions.go

```diff
@@ -32,29 +33,42 @@ type Function struct {
 	ArgTypes   []ValueType
 	Variadic   int
 	ReturnType ValueType
-	Call       func(ev *evaluator, args Expressions) Value
+
+	// vals is a list of the evaluated arguments for the function call.
+	//    For range vectors it will be a Matrix with one series, instant vectors a
+	//    Vector, scalars a Vector with one series whose value is the scalar
+	//    value,and nil for strings.
+	// args are the original arguments to the function, where you can access
+	//    matrixSelectors, vectorSelectors, and StringLiterals.
+	// ts is the evaluation timestamp.
```

Comment:
There is no `ts` argument. Should this be `enh.ts` instead?

Reply:
That's actually from a previous iteration of the PR, before ts was moved to enh.